lib/inspec/backend.rb

```diff
@@ -36,6 +36,13 @@ def inspect
           "Inspec::Backend::Class @transport=#{backend.class}"
         end
 
+        # Provide a shorthand to retrieve the inspec version from within a profile
+        #
+        # @return [String] inspec version
+        def version
```

Comment:
Great idea, this reminds me that we should make the meta-data accessible via `inspec.profile`. But lets do that later.